crates/context_aware_config/src/types.rs

```diff
@@ -0,0 +1,14 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct QueryFilters {
+    pub count: Option<i64>,
+    pub page: Option<i64>,
+}
+
+#[derive(Serialize, Debug, Clone, Deserialize)]
+pub struct PaginatedResponse<T> {
```

Comment:
should we move this to `superposition_types` in some new file for such common utility types as it might be useful for experimentation as well for list experiments

Reply:
Agreed, this template can be common throughout the repo